src/js/components/Diagram/StyledDiagram.js

```diff
@@ -1,12 +1,39 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { defaultProps } from '../../default-props';
+import { animationObjectStyle } from '../../utils/animations';
+
+const animationItemStyle = (item, theme) => {
+  if (typeof item === 'string') {
+    return animationObjectStyle({ type: item }, theme);
+  }
+  if (typeof item === 'object') {
+    return animationObjectStyle(item, theme);
+  }
+  return '';
+};
+
+const animationStyle = css`
+  ${props =>
+    props.animation === 'draw' || props.animation.type === 'draw'
+      ? css`
+          path {
+            stroke-dasharray: 500;
+            stroke-dashoffset: 500;
+            animation: ${animationItemStyle(props.animation, props.theme)};
+          }
+        `
+      : css`
+          animation: ${animationItemStyle(props.animation, props.theme)};
+        `}
+`;
 
 const StyledDiagram = styled.svg`
   max-width: 100%;
   width: 100%;
   height: 100%;
 
+  ${props => props.animation && animationStyle}
```

Comment:
Per the documentation in doc.js, it looks like animation values can be set as part of a connection. Is that so? If so, where is that getting applied?